youtube_dl/extractor/qqmusic.py

```diff
@@ -119,6 +119,12 @@ def _real_extract(self, url):
             'upload_date': publish_time,
             'creator': singer,
             'description': lrc_content,
+            'subtitles': {
+                'zh-CN': [{
```

Comment:
It's not an .lrc since it does not follow the format and not bound to time tags.